src/Tools/Microsoft.CodeAnalysis.Testing/Microsoft.CodeAnalysis.Codefix.Testing/CodeFixVerifier.cs

```diff
@@ -0,0 +1,45 @@
+﻿using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis.CodeFixes;
+using Microsoft.CodeAnalysis.Diagnostics;
+using Microsoft.CodeAnalysis.Analyzer.Testing;
+
+namespace Microsoft.CodeAnalysis.Codefix.Testing
+{
+    public class CodeFixVerifier<TAnalyzer, TCodeFix, TTest, TVerifier>
```

Comment:
❕ These types all need to be `static`. While this isn't a hard requirement for using the test framework with `using Verifier = ...` syntax, the alternative approach using `using static` doesn't consider static methods defined in base types, so allowing inheritance in these verifier types only leads to confusion.